docs/deployment/connect-scm.md

```diff
@@ -90,8 +90,8 @@ You have successfully connected an org in Semgrep AppSec Platform with an organi
 2. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login).
 3. Click **<i class="fa-solid fa-gear"></i> Settings > Source Code Managers > Add GitLab Cloud** and enter the personal access token generated into the **Access token** field.
 4. Enter your GitLab group's name into the **Name of your GitLab Group** field. If your repositories are organized in subgroups, you only need to provide the name of the top-level group.
-5. If you have multiple GitLab groups in your GitLab account, you need to create a source code manager per group. Repeat steps 1, 3-4 for each GitLab group.
-1. The GitLab groups are now listed under **Source Code organizations**. Click **Test connection** to verify that the new integration is installed correctly.
+5. Optional, but recommended: if you have multiple GitLab groups in your GitLab account, create a source code manager per group. Repeat steps 1, 3-4 for each GitLab group.
```

Comment:
```suggestion
5. Optional, but recommended: if you have multiple top-level GitLab groups in your GitLab account, create a source code manager per group. Repeat steps 1, 3-4 for each GitLab group.
```

I wonder if this is more clear, since we mention top-level groups in step 3 also,.

Reply:
I think what confuses me about the optional steps is, as a user, what would I gain from doing these optional steps?